<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeRenderer</class>
 <widget class="QWidget" name="ShapeRenderer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>171</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Shape</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ShapeWidget">
        <item>
         <property name="text">
          <string>Circle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Square</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="SizeWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="FillColorLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Fill Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FillColorButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="StrokeColorLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Stroke Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="StrokeColorButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Stroke Thickness</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="StrokeThiccWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Stroke Style</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="StrokeStyleWidget">
        <item>
         <property name="text">
          <string>Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dot</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
